server/team-validator-async.js

```diff
@@ -88,7 +89,7 @@ if (!PM.isParentProcess) {
 		});
 	}
 
-	global.Dex = require('../sim/dex').includeData();
+	global.Dex = (require(/** @type {any} */ ('../.sim-dist/dex'))).includeData();
```

Comment:
```suggestion
	global.Dex = require(/** @type {any} */ ('../.sim-dist/dex')).includeData();
```

I don't understand what's with the extra parens...

Reply:
Oh, this is from the other cast.

Reply:
Fixed.